.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -73,7 +74,8 @@ jobs:
         run: echo "GOOGLE_APPLICATION_CREDENTIALS=${{ github.workspace }}/.google_application_credentials" | Out-File -FilePath $env:GITHUB_ENV -Append
 
       - name: Install Swift toolchain
-        uses: compnerd/gha-setup-swift@main
+        # TODO(kendal): Merge this PR and revert to compnerd/gha-setup-swift before submitting.
```

Comment:
I'll resolve this before submitting.

Reply:
Yes please `compnerd/gha-setup-swift@main`

Reply:
Done!